apps/trading/components/select-market/select-market-columns.tsx

```diff
@@ -554,7 +554,14 @@ export const columnsPositionMarkets = (
     {
       kind: ColumnKind.Position,
       value: (
-        <p className={signedNumberCssClass(openVolume || '')}>{openVolume}</p>
+        // TODO add decimals to open volume
```

Comment:
chore (blocking): this comment seems irrelevant now

Reply:
removed.